pv_site_api/main.py

```diff
@@ -220,6 +221,7 @@ def post_site_info(site_info: PVSiteMetadata, session: Session = Depends(get_ses
 
 # get_pv_actual: the client can read pv data from the past
 @app.get("/sites/{site_uuid}/pv_actual", response_model=MultiplePVActual)
+@cache_response
```

Comment:
Maybe we should only cache the `many` version of the routes. Otherwise we could get funny things like having one route returning something different than the other.